src/bioseq/site_counting/site_counting.jl

```diff
@@ -89,20 +89,20 @@ for A in (DNAAlphabet, RNAAlphabet)
 
         # Match
         @inline function bp_chunk_count(::Type{Match}, ::Type{$A{4}}, a::UInt64, b::UInt64)
-            return count_zero_nibbles(a $ b)
+            return count_zero_nibbles(a ⊻ b)
         end
 
         @inline function bp_chunk_count(::Type{Match}, ::Type{$A{2}}, a::UInt64, b::UInt64)
-            return count_zero_bitpairs(a $ b)
+            return count_zero_bitpairs(a ⊻ b)
```

Comment:
`x $ y` is deprecated and now we should use `xor(x, y)` or `x ⊻ y`. You can use `\xor` to output `⊻` character on Julia 0.6.